### PR TITLE
fix: Removing oclif dev cli dependency and adding oclif core dependency

### PR DESCRIPTION
**Pull request by AsabuHere**
# Closes #
DII-551 Remove @oclif/dev-cli<2
A short description of what this PR does.
Removing oclif dev cli dependency and adding oclif core dependency 
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,8 +57,8 @@
   "devDependencies": {
     "@actions/core": "^1.0.0",
     "@actions/github": "^2.2.0",
-    "@oclif/dev-cli": "^1.22.2",
     "@oclif/test": "^1.2.6",
+    "@oclif/core": "1.13.3",
```
Comment (shrutiburman) [inline package.json:61]: oclif/core is already listed as a dependency.

File: package.json [hunk 2]
```diff
@@ -57,8 +57,8 @@
   "devDependencies": {
     "@actions/core": "^1.0.0",
     "@actions/github": "^2.2.0",
-    "@oclif/dev-cli": "^1.22.2",
```
Comment (shrutiburman) [inline package.json:60]: Was there any testing done to validate that dev-cli cmds still work via oclif/core ?

Comment (AsabuHere) [reply package.json:60]: yes, ran the command oclif-dev readme was executed, but didnt work.
The output was zsh: command not found: oclif-dev

**Comment by shrutiburman** [comment]
Closing as this is already resloved.

**Pull request closed without merging**